client/asset/btc/btc.go

```diff
@@ -920,7 +938,13 @@ func (btc *ExchangeWallet) split(value uint64, lots uint64, outputs []*output, i
 		return nil, false, fmt.Errorf("error creating change address: %w", err)
 	}
 
-	feeRate := btc.feeRateWithFallback(1) // these must fund swaps, so don't under-pay (is this an issue with no fundconf requirement?)
+	// This must fund swaps, so don't under-pay. TODO: get and use a fee rate
+	// from server, and have server check fee rate on unconf funding coins.
+	feeRate, err := btc.feeRate(1)
+	if err != nil {
+		// Fallback fee rate is NO GOOD here.
+		return nil, false, fmt.Errorf("unable to get optimal fee rate for pre-split transaction: %w", err)
+	}
```

Comment:
People are probably going to hit this, but we need this and a full solution is outside of the scope of this PR. I would just suggest that we get a more robust solution in before 0.2.

Reply:
Yeah, we do need a proper solution before next release.  I'm pretty sure people do hit this case now (and use the too-low fallback), but it's not clear how often it is BTC and for taker orders.

Reply:
Oh, but I think my reasoning for letting this ride is that they're aren't entirely blocked from placing orders, just placing orders with a pre-split txn. :)